audio_service/lib/audio_service.dart

```diff
@@ -879,7 +884,8 @@ class AudioService {
   static BaseCacheManager? _cacheManager;
 
   static late AudioServiceConfig _config;
-  static late AudioHandler _handler;
+  static AudioHandler get _handler => __handler!;
```

Comment:
It would be nice to avoid having `_handler` and `__handler`. I think an app should wait for `init()` to complete before attempting to use the handler. If the art fallback needs to be initialised before `init()` returns, consider making it a config parameter.

Reply:
This would mean we could not configure fallback before initializing the service, meaning that bad arts that are loaded on initialization could not fall back to configured default art.